sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/ComplexDataType.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+/**
+ * Simple DataType is used to create SimpleSearchField.
+ */
+public class ComplexDataType extends ExpandableStringEnum<ComplexDataType> {
```

Comment:
Why is this exposed? There's only 1 type of complex, or a collection of them. Whether you want a collection could just be a boolean (that's what I did for .NET).